CMakeLists.txt

```diff
@@ -125,6 +126,45 @@ include_directories(
     include/thirdparty
 )
 
+set(MAMBA_HEADERS
+    ${CMAKE_SOURCE_DIR}/include/activation.hpp
```

Comment:
To support including mamba as a CMake subproject (for example via FetchContent or git submodules) it could make sense to use `CMAKE_CURRENT_SOURCE_DIR` or `PROJECT_SOURCE_DIR` .

Reply:
Thanks for the comment! Yeah totally, mamba should also come with a generated CMake config file and all that so that find_package(mamba...) works.

Btw. we now have linux up to Rviz + desktop + MoveIt and RViz + desktop on OS X and RViz on Windows (I need to upload this) on the anaconda.org/robostack channel! We'll announce this tomorrow.